templates/iot/operator/020-Role-iot-operator.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
```

Comment:
This role should be added to the CSV in olm-manifest/src/main/resources/latest/... as well

Reply:
:+1: Done.